src/contractTest/java/uk/gov/hmcts/reform/em/npa/provider/NpaPactProviderTest.java

```diff
@@ -126,6 +127,11 @@ public void setupMarkupsForDocument() {
     }
 
 
+    @State("Markups exist for document f2cc4d79-d0f3-4b43-affe-535516370cdd and can be deleted")
```

Comment:
New state so we can just use "ID" instead od example UUID "f2cc4d79-d0f3-4b43-affe-535516370cdd".